source/Projectile.cpp

```diff
@@ -478,6 +490,13 @@ void Projectile::CheckConfused(const Ship &target)
 
 
 
+void Projectile::ConfusionDirection()
```

Comment:
This is a super confusing method name. We generally name getters this way, not... initializer helpers?

It might just be better to have this in the namespace (RandomDirection(), perhaps?), if it has to be a standalone function.

Reply:
Would it be better to just have `confusionDirection = Random::Int(2) ? -1 : 1;` where `ConfusionDirection()` is right now, instead of having a function for it?

Reply:
Yeah, that could be enough. Either that, or a namespace function.